### PR TITLE
bump: Go version to 1.20

### PR DESCRIPTION
**Pull request by hsinn0**
- Additional steps taken: `go mod tidy` & `go mod vendor`

[#185619304]

### CODE REVIEW

File: vendor/modules.txt [hunk 1]
```diff
@@ -25,9 +25,13 @@ github.com/google/go-cmp/cmp/internal/diff
 github.com/google/go-cmp/cmp/internal/flags
 github.com/google/go-cmp/cmp/internal/function
 github.com/google/go-cmp/cmp/internal/value
+# github.com/google/pprof v0.0.0-20230705174524-200ffdc848b8
+## explicit; go 1.19
```
Comment (swalchemist) [inline vendor/modules.txt:29]: Any idea what this means - the reference to go 1.19 here and go 1.18 and 1.19 below?

Comment (hsinn0) [reply vendor/modules.txt:29]: The file audo-updated by `go mod vendor`, which was required after cleaning it up with `go mod tidy`. It is a comment created by the tool, which I do not really care.

Comment (hsinn0) [reply vendor/modules.txt:29]: It appears that it is more than a comment. They call it annotation? See https://github.com/golang/go/issues/37946. Other than that I was not able to  find the reference on how to read the `vendor/modules.txt` file.

Comment (swalchemist) [reply vendor/modules.txt:29]: Ok, maybe the hash signs mean we're not really using those older versions.

**Review by swalchemist — APPROVED** [review]

**Comment by peterhaochen47** [comment]
superceded by https://github.com/cloudfoundry/credhub-acceptance-tests/pull/12

**Pull request closed without merging**